MockDuck/Sources/Utilities/SerializationUtils.swift

```diff
@@ -57,4 +57,11 @@ final class SerializationUtils {
 
         return fileName
     }
+
+    static func prefix(_ filename: String, withOrder order: Int) -> String {
+
+         var components = filename.components(separatedBy: "/")
+         let lastFilename = components.removeLast()
```

Comment:
`Array.removeLast()` assumes that the collection is not empty. Is that always the case here?